#! /bin/sh /usr/share/dpatch/dpatch-run
## 122-fix-cdf-file-syntax-errors.dpatch by  <kmccarty@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix some lines in CDF file descriptions that were accidentally
## DP: commented out by virtue of beginning with a '*' character.

@DPATCH@
diff -urNad cernlib-2006.dfsg.2~/src/packlib/cspack/programs/zftp/zftpcdf.cdf cernlib-2006.dfsg.2/src/packlib/cspack/programs/zftp/zftpcdf.cdf
--- cernlib-2006.dfsg.2~/src/packlib/cspack/programs/zftp/zftpcdf.cdf	1996-03-08 10:44:20.000000000 -0500
+++ cernlib-2006.dfsg.2/src/packlib/cspack/programs/zftp/zftpcdf.cdf	2007-04-21 18:56:29.000000000 -0400
@@ -388,8 +388,8 @@
 CHOPT 'CHOPT' C D=' '
 >Guidance
 Transfer all files matching the specified remote file name
-to the local system. The file name given may contain
-*, to match one or more characters, or %, to match a single
+to the local system. The file name given may contain *,
+to match one or more characters, or %, to match a single
 character.
 .
 By default the transfer is performed
@@ -397,9 +397,9 @@
 .
 e.g.
 .
-*.PAM         --> GETP
-*.CETA, *.CET --> GETB, LRECL=3600
-*.CMZ,  *.RZ  --> GETRZ
+ *.PAM         --> GETP
+ *.CETA, *.CET --> GETB, LRECL=3600
+ *.CMZ,  *.RZ  --> GETRZ
 .
 Options are passed to the appropriate transfer routine.
 >Action MGET
@@ -413,17 +413,18 @@
 CHOPT 'CHOPT' C D=' '
 >Guidance
 Transfer all files matching the specified local file name
-to the remote system. The file name given may contain
-*, to match one or more characters, or %, to match a single
+to the remote system. The file name given may contain *,
+to match one or more characters, or %, to match a single
 character.
+.
 By default the transfer is performed
 using PUTA, unless the file name has a known extension.
 .
 e.g.
 .
-*.PAM         --> GETP
-*.CETA, *.CET --> GETB, LRECL=3600
-*.CMZ,  *.RZ  --> GETRZ
+ *.PAM         --> GETP
+ *.CETA, *.CET --> GETB, LRECL=3600
+ *.CMZ,  *.RZ  --> GETRZ
 .
 e.g.
 .
diff -urNad cernlib-2006.dfsg.2~/src/packlib/fatmen/programs/fmkuip/fmcdf.cdf cernlib-2006.dfsg.2/src/packlib/fatmen/programs/fmkuip/fmcdf.cdf
--- cernlib-2006.dfsg.2~/src/packlib/fatmen/programs/fmkuip/fmcdf.cdf	1997-01-07 06:20:31.000000000 -0500
+++ cernlib-2006.dfsg.2/src/packlib/fatmen/programs/fmkuip/fmcdf.cdf	2007-04-21 18:58:52.000000000 -0400
@@ -324,8 +324,8 @@
 >Command SEARCH
 >Guidance
 Use the SEARCH command to print the generic names of files which
-match the specified criteria. Character fields may include the
-* or % wild cards.
+match the specified criteria. Character fields may include the *
+or % wild cards.
 .
 e.g. SEARCH * VID=I* # search current working directory for entries
 # with VID's beginning with I.
@@ -432,7 +432,7 @@
 If no wild-cards are present in the pathname, the characters /*
 are automatically appended. To zoom down a tree starting with
 .
-*ALLD, use ZOOM *ALLD/*
+ *ALLD, use ZOOM *ALLD/*
 .
 >MENU \FMDATA
 >Guidance
